### PR TITLE
deps: remove extraneous dependencies

### PR DESCRIPTION
**Pull request by rmg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,19 +21,14 @@
     "debug": "^2.2.0",
     "ibm_db": "^1.0.0",
     "loopback-connector": "^2.3.0",
-    "path": "^0.12.7",
```
Comment (Amir-61) [inline package.json]: path is already being used here: https://github.com/strongloop/loopback-ibmdb/blob/0612370804879a879e3593c718ceb3c0289bbe74/lib/globalize.js#L6

**Comment by rmg** [comment]
`path` is part of node core and including the module version as a dependency anywhere will change what the entire application uses because npm will hoist it up to the top level. This is very bad behaviour.

**Comment by superkhau** [comment]
Please backport it to `loopback-2.x` too -- see https://github.com/strongloop-internal/loopback-knowledge-base/blob/master/backporting.md

**Comment by superkhau** [comment]
>Put path back in deps as lib/globalize uses it as stated by @Amir-61, otherwise LGTM.

@rmg explained on Slack, we should be using path from Node core instead. :ship:

**Comment by Amir-61** [comment]
LGTM

**Pull request merged**